std/file.d

```diff
@@ -3094,7 +3094,7 @@ void copy(RF, RT)(auto ref RF from, auto ref RT to, PreserveAttributes preserve
 
 unittest // issue 15319
 {
-    import std.path: dirEntries;
+    import std.file : dirEntries;
```

Comment:
`dirEntries` is not visible via `std.path` because it privately imports `std.file`.